frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -667,6 +672,24 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [OCS_FLAG],
     },
   },
+  // In case Noobaa is deployed without OCS Operator
+  {
+    type: 'NavItem/Href',
+    properties: {
+      id: 'ocsdashboards',
+      section: 'storage',
+      insertBefore: 'persistentvolumes',
+      componentProps: {
+        // t('ceph-storage-plugin~Overview')
+        name: '%ceph-storage-plugin~Overview%',
+        href: '/ocs-dashboards',
```

Comment:
why we cant keep just overview ? aligning with component visible on UI.
```suggestion
        href: '/overview',
```

Reply:
will clash with the cluster overivew.

Reply:
I meant `/storage/overview` or `/storage/overview/ocs`. This route seems better IMO.
Also it will be similar to what `monitroing` section does for their nav items - `/monitoring/dashboards/`

Reply:
I can live with ocs-dashboards for now.We shoudl revisit this when we want to make the section better by standardizing routes for OBC, PVC, VS, SC etc..